{
  throw new RuntimeException("This is an error created by the application");
}
